generator/test_case_generator.go

```diff
@@ -87,6 +87,13 @@ var testTypeToTestConfig = map[string][]testConfig{
 	"ec2_acceptance": {
 		{"./test/acceptance", ""},
 	},
+	"ec2_userdata": {
+		{"./test/cloudwatchlogs", ""},
+		{"./test/metric_value_benchmark", ""},
```

Comment:
Initially I was running all these tests to imitate the internal userdata tests which just uses base integration test to test logs and bunch of metrics. These do not install agent outside of userdata since the install behavior is determined in each test's main.tf file and userdata test has its own unique main.tf file, so the agent installed is the one from userdata.

However I agree that having these tests are redundant and for the purpose of testing userdata, just a simple CPU config test should suffice so I removed them.